README.md

```diff
@@ -8,6 +8,9 @@ and lots of other gunk. By using a combination of various techniques and
 detection mechanisms, the validator is capable of being both efficient as well
 as thorough.
 
+:warning: :warning: :fire: This tool is being deprecated and going away eventually. ::warning: :warning: :fire:
+It is still used on addons.mozilla.org for legacy add-ons but once support for updates for legacy add-ons goes away from AMO this tool will be archived. Please don't use it.
```

Comment:
I'd delete `Please don't use it.`.

Also, move that section right below the headline?

Otherwise, looks great!